### ISSUE TITLE
[Bug]: AdminerEvo shown for unsecured sites

### ISSUE THREAD

**Issue by emfMaddin** [beyondcode/herd-community#900]
### Platform

macOS

### Operating system version

macOS Sonoma 14.5

### System architecture

ARM64 (M1, M2, etc)

### Herd Version

1.9.1

### Bug description

Accessing an unsecured site via http:// shows Adminer Evo on Mac. Reading the changelog for mac does not even mention that Adminer was or is included with Laravel Herd. Via google I found the information that on windows adminer was included with version 1.8.0. 

This confused me a lot today and i somehow want to be able to use sites unsecured via http:// on my local machine.

**Comment by mpociot** [comment]
Can you tell me how you are able to reproduce this? 
It sounds like this happens for *every* unsecured site for you?

I just tried this via:

```
laravel new adminer-evo-issue-test
```

Once that was done, I accessed `http://adminer-evo-issue-test.test` in the browser and saw the Laravel site.

If you want to do some screen sharing, feel free to join this Google meet: https://meet.google.com/cdd-ijhi-bfb

**Comment by emfMaddin** [comment]
Thanks for quick response. Yes, it happens for every unsecured page here on my machine. 
I set up a new laravel page like you did, but it shows adminer evo here. 

I'll join the google meet in a second.

**Comment by mpociot** [comment]
@emfMaddin Could you tell me again what you chose as your updated TLD?

**Comment by emfMaddin** [comment]
@mpociot sure, its "testms"

**Comment by mpociot** [comment]
I just tried to replicate this in a fresh macOS VM by using the (no longer officially supported) "herd tld" command and I can't reproduce this issue as all sites seem to load just fine.

<img width="1016" alt="image" src="https://github.com/user-attachments/assets/170ba9dc-0ce5-41c7-8efc-5da8fbb9af52">

**Comment by emfMaddin** [comment]
Okay. I'll have a look if I can find out whats going on here.

**Comment by emfMaddin** [comment]
I checked the full nginx config with this command:
```
/Applications/Herd.app/Contents/Resources/nginx-arm64 -c "/Users/mschiessl/Library/Application Support/Herd/config/nginx/nginx.conf" -T > fullconf.txt
```

and figured out the following points:

1. for some sites there is no nginx config - probaly because there is no conf file /Users/[USERNAME]/Library/Application Support/Herd/config/valet/Nginx ?
Looks like there are only conf files for secured sites, is this as expected? Securing a site brings up the conf, after unsecuring the same site the conf is gone again.

2. for some old pages there are still config files with the previous tld "test"
Removing these did not help. 

Is there a way to re-generate the nginx config for all sites?

**Comment by mpociot** [comment]
That's true - Herd (and Valet) only create custom nginx configuration files when you either secure or isolate a site.
Otherwise, all requests to `some-domain.test` get redirected to the default nginx server which is defined in your `herd.conf` file.
This acts as a "catch-all" server and then Herd figures out which driver to use for the site and how to serve it.

That's why the `herd.conf` contains:

```
server {
    listen 127.0.0.1:80 default_server; // <--

    ...
```

You can force Herd to re-create the `herd.conf` file, by simply removing (or renaming) it.

**Comment by sebasvisser** [comment]
Had the same problem on fresh install, migrating from Valet.

renaming herd.conf file to old-herd.conf. And restarting Herd fixed it for me.

**Comment by Gnative** [comment]
I'm having this issue now too. I change my herd.conf to open the :80 port for testing on my machine as it allows mobile devices to access my herd sites for API endpoints via my local IP. This is now unusable after the last update and I'm not sure how to resolve it. 

```
server {
    listen 80 default_server;
```

I've always followed this documentation as explained in this comment - https://github.com/beyondcode/herd-community/issues/276 

Can you give me an alternative solution ?
Thanks

**Comment by Craytor** [comment]
I recently upgraded to the latest Herd on MacOS and am having this issue as well.

> Had the same problem on fresh install, migrating from Valet.
> 
> renaming herd.conf file to old-herd.conf. And restarting Herd fixed it for me.

I can confirm that I also had this issue. I upgraded to the latest Herd on MacOS and saw AdminerEvo on all unsecured sites. Secured sites worked fine, however. I renamed the config file `~/Library/Application Support/Herd/config/nginx/herd.conf` to `~/Library/Application Support/Herd/config/nginx/herd_old.conf` and restarted Herd. Everything is working as expected now.
